db/seeds.rb

```diff
@@ -1,504 +1,504 @@
- extract_potential: 1.032, alpha_acid: nil, attenuation: nil)
-Ingredient.create(name:'Cara-Pils/Dextrine', type_class: 'Grain',
- extract_potential: 1.033, alpha_acid: nil, attenuation: nil)
-Ingredient.create(name:'Caramel/Crystal Malt – 10L', type_class: 'Grain',
- extract_potential: 1.035, alpha_acid: nil, attenuation: nil)
-Ingredient.create(name:'Caramel/Crystal Malt – 20L', type_class: 'Grain',
- extract_potential: 1.035, alpha_acid: nil, attenuation: nil)
-Ingredient.create(name:'Caramel/Crystal Malt – 30L', type_class: 'Grain',
- extract_potential: 1.035, alpha_acid: nil, attenuation: nil)
-Ingredient.create(name:'Caramel/Crystal Malt – 40L', type_class: 'Grain',
- extract_potential: 1.034, alpha_acid: nil, attenuation: nil)
-Ingredient.create(name:'Caramel/Crystal Malt – 60L', type_class: 'Grain',
- extract_potential: 1.034, alpha_acid: nil, attenuation: nil)
-Ingredient.create(name:'Caramel/Crystal Malt – 80L', type_class: 'Grain',
- extract_potential: 1.034, alpha_acid: nil, attenuation: nil)
-Ingredient.create(name:'Caramel/Crystal Malt -120L', type_class: 'Grain',
- extract_potential: 1.033, alpha_acid: nil, attenuation: nil)
-Ingredient.create(name:'Chocolate Malt', type_class: 'Grain',
- extract_potential: 1.028, alpha_acid: nil, attenuation: nil)
-Ingredient.create(name:'Melanoiden Malt', type_class: 'Grain',
- extract_potential: 1.037, alpha_acid: nil, attenuation: nil )
-Ingredient.create(name:'Munich Malt – 10L', type_class: 'Grain',
- extract_potential: 1.035, alpha_acid: nil, attenuation: nil )
-Ingredient.create(name:'Munich Malt – 20L', type_class: 'Grain',
- extract_potential: 1.035, alpha_acid: nil, attenuation: nil )
-Ingredient.create(name:'Pale Malt (6 Row)', type_class: 'Grain',
- extract_potential: 1.035, alpha_acid: nil, attenuation: nil)
-Ingredient.create(name:'Pilsner (2 Row)', type_class: 'Grain',
- extract_potential: 1.036, alpha_acid: nil, attenuation: nil)
-Ingredient.create(name:'Smoked Malt', type_class: 'Grain',
- extract_potential: 1.037, alpha_acid: nil, attenuation: nil)
-Ingredient.create(name:'Special Roast', type_class: 'Grain',
- extract_potential: 1.033, alpha_acid: nil, attenuation: nil)
-Ingredient.create(name:'Toasted Malt', type_class: 'Grain',
- extract_potential: 1.033, alpha_acid: nil, attenuation: nil)
-Ingredient.create(name:'Victory Malt', type_class: 'Grain',
- extract_potential: 1.034, alpha_acid: nil, attenuation: nil)
-Ingredient.create(name:'Vienna Malt', type_class: 'Grain',
- extract_potential: 1.036, alpha_acid: nil, attenuation: nil)
+Ingredient.create(name:'Acid Malt', sub_type: 'Grain',
+ extract_potential: 1.027, alpha_acid: nil, attenuation: nil, main_class: "Grain")
+Ingredient.create(name:'Amber Malt', sub_type: 'Grain',
+ extract_potential: 1.035, alpha_acid: nil, attenuation: nil, main_class: "Grain")
+Ingredient.create(name:'Aromatic Malt', sub_type: 'Grain',
+ extract_potential: 1.036, alpha_acid: nil, attenuation: nil, main_class: "Grain")
```

Comment:
Line is too long. [82/80]<br>Align the elements of a hash literal if they span more than one line.